builtin/providers/postgresql/GNUmakefile

```diff
@@ -0,0 +1,28 @@
+POSTGRES?=/opt/local/lib/postgresql96/bin/postgres
```

Comment:
I think it's worth adding a comment to the top of this file stating the intended use, or use the "self-documenting" structure and make the `help` target be the default.